src/node-server/openfin-config/src/config/launcher.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
```

Comment:
There is a lot of commonality between app.ts and launcher.ts - can we be smarter about sharing between the two?

Reply:
Nice, I'll create a base these can extend.